### ISSUE TITLE
Thumbnails for "Continue watching don't show"

### ISSUE THREAD

**Issue by f1reflyyyylmao** [streamaserver/streama#685]
### Issue description
When i begin watching a series/movie that was added with the themoviedb api it doesn't show a thumbnail in the "continue watching" section. Thumbnails in other sections work fine. Self-added thumbnails show up, too. The server and client are both online, streama doesn't report errors. I set up streama to use a mariadb which appears to work just fine. I don't think this is related to #455 .

### Steps to Reproduce

1. Launch streama
2. Add a valid themoviedb api key
3. Add a series/movie with data from themoviedb
4. Begin watching it (port 8080) and go back to the menu

### Expected Behaviour

Streama should show thumbnails for series/movies added with the api

### Actual Behaviour

They're missing

### Environment Information

- **Operating System**: Arch Linux x64
- **Streama version**: v1.6.0-RC9

### Picture
![2018-11-09-232805_1144x843_scrot](https://user-images.githubusercontent.com/17353799/48291838-269f9080-e478-11e8-9e84-2dcaa5cdce70.png)

**Comment by jguerinet** [comment]
+1. This is also the case for the "Recommended for You" section.

**Comment by wanderfyx** [comment]
Hey, @f1reflyyyylmao  add me in discord: Wander#2568
for fix

**Comment by wanderfyx** [comment]
Please send me your discord name#tag please.

**Comment by f1reflyyyylmao** [comment]
I don't have a discord account and, duo to their privacy policy I won't create one. Why don't you write your fix in the wiki or better create a pull request to fix it?

**Comment by HannahRy** [comment]
@wanderfyx If you have a fix, discord isn't the best place for it. Please update this issue or create a PR.

**Comment by wanderfyx** [comment]
I do not know very well how I did the correction, but I remember doing something that out of nothing all miniature came back to appear.

**Comment by f1reflyyyylmao** [comment]
Just try to describe as good as you can what you did to make it work again. If you're uncomfortable with that I'm sure we can find a way to speak and I'll write it up if it works.

**Comment by f1reflyyyylmao** [comment]
After upgrading to the newest streama version this issue is resolved for me, @jguerinet please verify if it works for you too

**Comment by jguerinet** [comment]
RC10 fixes the thumbnails in the "Continue Watching" section, but the ones in the "Recommendations for You" are unfortunately still missing.

**Comment by f1reflyyyylmao** [comment]
How can i make streama show the "recommended for you" section? It's not being displayed on my dash..

**Comment by jguerinet** [comment]
In the "Hidden dash sections" within the admin settings, do you have `recommends`? If so, remove that and save.

**Comment by f1reflyyyylmao** [comment]
No, my "hidden dash sections" is empty. However, i just noticed that i get a ton of sql errors because a foreign key restraint appears to fail, so there's probably something broken with my sql database. I'll delete the whole thing and report again when i managed to get it to run without errors

**Comment by f1reflyyyylmao** [comment]
After resetting my database and re-adding some movies, theres still no section called "recommended for you". I didn't find anything about it in the docs either. Is there a place i can read about this feature?

**Comment by jguerinet** [comment]
[This](https://github.com/streamaserver/streama/issues/53) is the original issue. After looking it up, it looks like the Recommendations section only shows up if you  have chosen a favorite genre (Profile icon in top right corner -> user settings -> Choose a genre that has at least 1 video associated to it). 
Make sure to refresh the browser once you do this.

**Comment by f1reflyyyylmao** [comment]
After setting some favorite genres i still don't get any recomendations despite having some movies for most of them in my library

**Comment by jguerinet** [comment]
Hm, I'm not sure what is happening then. Maybe @dularion can shed some light on why the recommended section isn't showing up?

**Comment by f1reflyyyylmao** [comment]
I already fixed it in my reply, i guess github just didn't send another email to you? It works now and I can confirm that the thumbnails are missing.

**Comment by jguerinet** [comment]
Yeah, I don't believe GH sends emails for edited replies. Good to know that I'm not the only one!
